core-build/gulp-core-build-typescript/src/ApiExtractorTask.ts

```diff
@@ -68,6 +68,38 @@ export interface IApiExtractorTaskConfig {
    * @beta
    */
   dtsRollupTrimming: boolean;
+
+  /**
+   * This setting is only used if "trimming" is true.
+   * It indicates the folder where "npm publish" will be run for an internal release.
+   * The default value is "./dist/internal".
+   *
+   * @beta
+   * An internal release will contain all definitions that are reachable from the entry point.
+   */
+  publishFolderForInternal?: string;
+
+  /**
+   * This setting is only used if "trimming" is true.
+   * It indicates the folder where "npm publish" will be run for a beta release.
+   * The default value is "./dist/beta".
+   *
+   * @beta
+   * A beta release will contain all definitions that are reachable from the entry point,
+   * except definitions marked as \@alpha or \@internal.
+   */
+  publishFolderForBeta?: string;
+
+  /**
+   * This setting is only used if "trimming" is true.
```

Comment:
>true [](start = 48, length = 4)

As part of this PR, could you fix the 3 typos that we found in the [api-extractor.schema.json file](https://github.com/Microsoft/web-build-tools/blob/77793c7005b36ce5cf4ad1fbb0ea0a8f32b77c50/apps/api-extractor/src/extractor/api-extractor.schema.json#L148).  It should say "is true" instead of "is false".  That will make it consistent with this file.